.gitignore

```diff
@@ -164,5 +164,8 @@ bin/
 main/java/frc/robot/common/
 
 BuildConstants.java
+RobotContainer.java
+Robot.java
+Main.java
```

Comment:
Why would you want to ignore these files? Don't you want changes in them to be uploaded to Git?

Reply:
These are just for testing the simulation of the swerve, common is a library so if it has these files it creates problems for projects using it.

Reply:
OK true, I forgot it.